metabase/metabase
# Fix - Data source picker sources not refreshed after archiving a question

Fixes #32252

Depends on #45624

### How to verify

1. Start a fresh Metabase instance
2. New > Question
3. Verify that "Saved questions" tab **is present** in the data picker modal
4. Go to collections and archive the "Examples" collection
5. New > Question
6. Verify that "Saved questions" tab **is not present** in the data picker modal

Stress test: https://github.com/metabase/metabase/actions/runs/9953607328
